Rocket.Chat/Controllers/Chat/ChatControllerMessageActions.swift

```diff
@@ -24,15 +25,24 @@ extension ChatViewController {
                 presenter.sourceRect = view.bounds
             }
 
+            controller.emojiPicked = { emoji in
+                MessageManager.react(message, emoji: emoji, completion: { _ in })
+            }
+
+            controller.customEmojis = {
+                guard let emojis = Realm.shared?.objects(CustomEmoji.self) else { return [] }
```

Comment:
We could avoid importing `RealmSwift` here by just having a method in `CustomEmoji` that returns the objects as Emojis. What do you think?